client/src/components/common/Footer/index.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faEnvelope,
+  faMapMarker,
+  faPhone,
+} from '@fortawesome/free-solid-svg-icons';
+import {
+  faTwitter,
+  faInstagram,
+  faFacebookSquare,
+  faBehance,
+  faSkype,
+  faPinterest,
+} from '@fortawesome/free-brands-svg-icons';
+import logo from '../../../assets/images/berwaz.png';
+import './style.css';
+
+const Footer = () => {
+  return (
+    <div className="main-container">
+      <div className="footer-container">
+        <div className="follow-container">
+          <p>تابعنا على</p>
+          <div className="follow-icons">
+            <a
+              href="https://twitter.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon icon={faTwitter} color="#636D73" size="2x" />
```

Comment:
I can see that you wrote `color="#636D73"` like 9 times, but what if you want to change this main color ?
you will edit it manually for all the 9 cases, I think you can define a const with the requested color value and call for it whenever you want, and if you want to change the color you will change it only once :D